restapi/src/test/java/io/stargate/web/docsapi/service/util/DocsApiUtilsTest.java

```diff
@@ -56,42 +56,42 @@
     public void happyPath() {
       int index = RandomUtils.nextInt(100, 999);
 
-      String result = DocsApiUtils.convertArrayPath(String.format("[%d]", index));
+      String result = DocsApiUtils.convertArrayPath(String.format("[%d]", index), 999999);
```

Comment:
can you just add one test that simulates array index higher that given max? and verify correct error being returned?